MM-15312 - Updating tooltip UX guidelines (#2682)
mattermost/docs: source/developer/fx-guidelines.rst

|
---------------------------

> Display: Tooltips

Reviewer: Propose moving below `Display: Instructions` to keep the design guidelines section in alphabetical order.

Author: Done.

---------------------------

> Tooltips text can be title case or sentence case.

Reviewer: ```suggestion
Tooltip text may be title case or sentence case depending on their context.
```

Author: Shouldn't this be, depending on its context, as a "Tooltip" is singular.
Or we can use "Tooltips" and then use "their".

Author: @jasonblais ^

Reviewer: @asaadmahmood Yes, good catch. 

```
Tooltip text may be title case or sentence case depending on its context.
```

Author: Makes sense.

> - Tooltips will be title case if the tooltip is a noun, e.g. Pinned Posts, or Flagged Posts.

Reviewer: ```suggestion
- Tooltips should be title case if the tooltip is a noun, e.g. "Pinned Posts", or "Flagged Posts".
```

> - Tooltips will be sentence case if they are announcing an action, e.g. Flag for follow up, or Create new private channel.

Reviewer: ```suggestion
- Tooltips should be sentence case if they are declaring an action, e.g. "Flag for follow up", or "Create new private channel".
```
